amazon/no-nap/entrypoint.sh

```diff
@@ -29,6 +29,16 @@ nginx -g "daemon off;" &
 
 nginx_pid=$!
 
+wait_workers()
+{
+    while ! ps aux | grep '[n]ginx: worker process' >/dev/null 2>&1; do
```

Comment:
Maybe it would be more readable to replace that with something like :
```
pgrep -xf "nginx: worker process" > /dev/null
```
Also, why in `grep` are you using `[n]` instead of `n`?

Reply:
good suggestion, theoretically `pgrep` might be missing in some "slim" images. 

`"[n]"` is the easy regex way to exclude the extra line like that:
```
ubuntu    1188  0.0  0.0  16796  2216 pts/0    S+   01:13   0:00 grep --color=auto nginx: worker process
```

Reply:
updated to `pgrep -f`, `-xf` is not working on ubuntu.